README.md

```diff
@@ -141,17 +136,48 @@ experimentation while writing code that is production-ready from the get-go.
 - 🛫 ZenML enables MLOps infrastructure experts to define, deploy, and manage
 sophisticated production environments that are easy to use for colleagues.
 
-![The long journey from experimentation to production.](/docs/book/.gitbook/assets/intro-zenml-overview.png)
-
-ZenML provides a user-friendly syntax designed for ML workflows, compatible with
-any cloud or tool. It enables centralized pipeline management, enabling
-developers to write code once and effortlessly deploy it to various
-infrastructures.
-
 <div align="center">
-    <img src="docs/book/.gitbook/assets/overview.gif">
+  <img width="60%" src="/docs/book/.gitbook/assets/zenml-hero.png" alt="ZenML Hero"/>
 </div>
 
+ZenML optimizes your entire machine learning lifecycle - you'll be 80% faster to production, generate 3x more models, and massively reduce your cloud spend.
```

Comment:
I think these numbers can be very situational and it feels a bit off displaying them without any resources to back it up.